bindings/cs/MongoDB.Libmongocrypt/MongoDB.Libmongocrypt.csproj

```diff
@@ -74,7 +74,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Content Include="MongoDB.Libmongocrypt.targets">
+    <Content Include="MongoDB.Libmongocrypt.props">
```

Comment:
I have no idea what this change means.

Reply:
Oh I see this is referencing a file YOU added. But why add this is a separate file?

Reply:
Done.